terragrunt/aws/glue/etl.tf

```diff
@@ -51,6 +51,7 @@ resource "aws_glue_trigger" "platform_gc_forms_job" {
   name     = "Platform / GC Forms"
   schedule = "cron(00 2 * * ? *)" # 2am UTC every day
   type     = "SCHEDULED"
+  enabled  = false
```

Comment:
This is temporarily disabled while we wait for the prod data replication to start.